build_tools/pkgci/external_test_suite/models_gpu_rocm_gfx90a.json

```diff
@@ -23,7 +23,7 @@
     // TODO(#17874): error: a handle passed as operand #0 and consumed by this operation points to a payload entity more than once
     "sharktank/llama/open-llama-3b-v2-f16",
 
-    "sharktank/punet/fp16",
```

Comment:
Why are we dropping this test?

Reply:
The test was listed in `expected_compile_failures`, but now it is newly passing, so it gets removed from the XFAIL list.

The config implicitly includes all tests that are discovered at the moment. That lets new tests be picked up without needing to add them to config files. Might change that in favor of a list like
```json
"expected_test_results": [
  {"model_foo": "pass"},
  {"model_bar": "fail_compile"},
  {"model_baz": "fail_run"},
]
```

Reply:
+1, I was also confused on that, but further reading seems to indicate that this test CI workflow discovers from another repo (https://github.com/nod-ai/SHARK-TestSuite/tree/da6b62182631e30228989f1e1e90085d5629931d/iree_tests)

Reply:
Docs: https://iree.dev/developers/general/testing-guide/#external-test-suite

Reply:
oh 🤦 I should have looked 5 lines up, I didn't realize this was an xfail.